src/kvm_bindings_versionize/src/x86/bindings/tests.rs

```diff
@@ -0,0 +1,2096 @@
+//! Import layout tests from upstream to ensure our struct definitions match.
+
+use super::*;
+
+#[test]
+fn bindgen_test_layout_kvm_pic_state() {
```

Comment:
Since these structs are shadowing the upstream ones, I think the layout tests for them should compare _size_, _alignment_, etc against the upstream struct instead of hardcoded values.
This would improve on the mechanisms to keep them in sync.

Reply:
The exact layout tests are generated by upstream, but they're now disconnected from upstream.

If we upgrade used upstream version, the values hardcoded here (generated with previous upstream version) might be stale.

If these tests would instead compare local structs against imported/upstream structs, the code here would not get stale and would work correctly for any upstream version.